packages/block-editor/src/store/test/selectors.js

```diff
@@ -2343,4 +2344,83 @@ describe( 'selectors', () => {
 			} );
 		} );
 	} );
+
+	describe( 'getLowestCommonAncestorWithSelectedBlock', () => {
+		const blocks = {
+			order: {
+				'': [ 'a', 'b' ],
+				a: [ 'c', 'd' ],
+				d: [ 'e' ],
+				f: [ 'b' ],
```

Comment:
This doesn't seem to match the `parents` below. Was this meant to be `b: [ 'f' ]`?